python/whylogs/core/schema.py

```diff
@@ -56,9 +56,9 @@ class DatasetSchema:
     """
 
     types: Dict[str, Any] = field(default_factory=dict)
-    default_configs: MetricConfig = MetricConfig()
-    type_mapper: TypeMapper = StandardTypeMapper()
-    resolvers: Resolver = StandardResolver()
+    default_configs: MetricConfig = field(default_factory=MetricConfig)
```

Comment:
oof, this doesn't seem very readable especially as we make more custom ones like @FelipeAdachi example. Could this be iterated on?